edx_proctoring/static/proctoring/templates/student-proctored-exam-attempts-grouped.underscore

```diff
@@ -0,0 +1,240 @@
+<div class="wrapper-content wrapper">
```

Comment:
this file contains a lot of the same code that's in student-proctored-exam-attempts.underscore, but I wanted to separate it out into its own file so its easier to modify and deprecate code frontend code related to this feature change